pynet/models/spherical/unet.py

```diff
@@ -285,7 +287,8 @@ class UpBlock(nn.Module):
     upconv => (conv => BN => ReLU) * 2
     """
     def __init__(self, conv_layer, in_ch, out_ch, conv_neigh_indices,
-                 neigh_indices, up_neigh_indices, down_indices, up_mode):
+                 neigh_indices, up_neigh_indices, down_indices, up_mode,
+                 merge_mode="concat"):
         """ Init.
 
```

Comment:
Please update docstring with the new parameter.
No need to set a default value.